README.md

```diff
@@ -1 +1,35 @@
-# Hello-world
\ No newline at end of file
+Hello Microverse
+The project is about Microverse courses.
+
+Built With
+Javascript, CSS, HTML
+
+Prerequisites
```

Comment:
Good job in using the template, however your project does not use the correct markup to achieve the same results as the template does. This  [markup cheatcheat](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) can assist you have the best use of markup

This is your current readme file:
<img width="887" alt="Screenshot 2022-09-21 at 02 03 35" src="https://user-images.githubusercontent.com/48546992/191380098-b20d540a-cd65-4384-8866-9c8436505ef1.png">

This is the [readme](https://github.com/microverseinc/readme-template/blob/master/README.md)template provided by microverse
<img width="1211" alt="Screenshot 2022-09-21 at 02 03 57" src="https://user-images.githubusercontent.com/48546992/191380304-77b48909-8318-456c-a40d-65ca0f9d6990.png">
The template provided by microverse is very well formatted using markdown. This provides users, potential recruiters a better feel of your project.

I highly recommend that you copy the template. Click on the edit pen ✏️  on the template provided or the copy functionality  (two boxes)   and just copy the template, then edit it to suit your needs. 
<img width="1197" alt="Screenshot 2022-09-21 at 02 03 57" src="https://user-images.githubusercontent.com/48546992/191381073-86b16cbe-103d-445c-9367-d6b632df3032.png">